components/Home/Hero/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { downloadUrlByOS } from '@/util/downloadUrlByOS';
+import Link from 'next/link';
+import type { FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useDetectOS } from '@/hooks/useDetectOS';
+import styles from './index.module.scss';
+
+type HeroProps = {
+  title: string;
+  subTitle?: string;
+  ltsVersion: string;
+  currentVersion: string;
+};
+
+const Hero: FC<HeroProps> = ({
+  title,
+  subTitle,
+  ltsVersion,
```

Comment:
I think that this is cleaner:
```js
const { getReleaseByStatus } = useNodeReleases();
const [lts, current] = useMemo(
  () => [getReleaseByStatus('Active LTS'), getReleaseByStatus('Current')],
  [getReleaseByStatus]
);
```
And use `lts.version` and `current.version` rather than props drilling these 2 values since Node releases data is available in all components in the tree.

Reply:
@HinataKah0 I thought of this approach but could not proceed further as the file "node-releases-data.json" is empty